build.sbt

```diff
@@ -81,7 +81,7 @@ val commonSettings = Seq(
     publishArtifacts,
     setNextVersion,
     commitNextVersion,
-    releaseStepCommand("sonatypeReleaseAll"),
+    releaseStepCommand("sonatypeReleaseBundle"),
```

Comment:
Is this because `sonatypeReleaseAll` has been removed and replaced by `sonatypeReleaseBundle` in `3.7`?

Reply:
I undid this after all, I may have misunderstood how it's supposed to work.

Reply:
Cue the complete lack of documenttation.